bindings/kotlin/examples/PrepareTransferObjectsOffline.kt

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import iota_sdk.*
+import kotlinx.coroutines.runBlocking
+
+fun main() = runBlocking {
+    try {
+        val client = GraphQlClient.newDevnet()
+
+        val fromAddress =
+                Address.fromHex(
+                        "0x611830d3641a68f94a690dcc25d1f4b0dac948325ac18f6dd32564371735f32c"
+                )
+        val toAddress =
+                Address.fromHex(
+                        "0x0000a4984bd495d4346fa208ddff4f5d5e5ad48c21dec631ddebc99809f16900"
+                )
+        val objects =
+                listOf(
+                        ObjectId.fromHex(
+                                "0xd04077fe3b6fad13b3d4ed0d535b7ca92afcac8f0f2a0e0925fb9f4f0b30c699"
+                        ),
+                        ObjectId.fromHex(
+                                "0x0b0270ee9d27da0db09651e5f7338dfa32c7ee6441ccefa1f6e305735bcfc7ab"
+                        ),
+                        ObjectId.fromHex(
+                                "0x8ef4259fa2a3499826fa4b8aebeb1d8e478cf5397d05361c96438940b43d28c9"
+                        )
+                )
+        val objsToTransfer =
+                objects.map {
+                    val obj = client.`object`(it)
```

Comment:
Is `object` a keyword?

Reply:
yep

Reply:
no, but what would you prefer?

Reply:
I'm pretty sure there is not

Reply:
Sure but it's really weird and confusing. I assume that kotlin users know their language is shit and how to avoid pitfalls like this, just like how rust users know they can't use fields named `type`